.github/workflows/mlc_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "retryOn429": true,
+  "retryCount": 3,
+  "fallbackRetryDelay": "45s"
```

Comment:
45 s = 45 seconds? Seems like too much.

Maybe something like 5s would be enough?

Reply:
Yes "45s" = 45 seconds. Github rate limits quite aggressively, and [most open source packages using this Action](https://sourcegraph.com/search?q=context:global+%22fallbackRetryDelay%22:+lang:json&patternType=literal) use "30s", but we can decrease it to 10s and increase it if we continue receiving 429 errors.

Reply:
Let's see if its enough :)